src/components/Balance.js

```diff
@@ -1,23 +1,28 @@
-import React, { useState } from "react";
+import React from "react";
 import "./Balance.css";
 
-const Balance = (props) => {
-  const [alternateCurrency] = useState("USD");
-  const { total, currencies } = props;
+const Balance = ({ total, currencies, setFromGBPto, convertedValue, fromGBPto }) => {
   return (
     <div className="Balance">
       <h2 className="Balance-title">
         Your account balance is
         <span className="Balance-total">£{total}</span>
       </h2>
       <div className="Balance-alt">
-        Your balance is ??? in
-        <select defaultValue={alternateCurrency}>
```

Comment:
Where is the closing tag for this select? 🤔

Reply:
I will fix it now